hypershift-operator/controllers/externalinfracluster_controller.go

```diff
@@ -24,18 +24,18 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-type GuestClusterReconciler struct {
+type ExternalInfraClusterReconciler struct {
```

Comment:
The event recorder in line 53 should be for "external-infra-cluster-controller" ... https://github.com/openshift-hive/hypershift/pull/31/files#diff-2234353920a980ab201b2fc06e4ef30d221189c9655e2f8922e5877cff1b5325L53